lib/6to5/generation/position.js

```diff
@@ -1,33 +1,27 @@
 module.exports = Position;
 
-var _ = require("lodash");
-
 function Position() {
   this.line = 1;
   this.column = 0;
 }
 
 Position.prototype.push = function (str) {
-  var self = this;
-
-  _.each(str, function (cha) {
-    if (cha === "\n") {
-      self.line++;
-      self.column = 0;
+  for (var i = 0; i < str.length; i++) {
```

Comment:
I'm not sure if it makes a difference, but does caching the length here help? i.e. `for (var i = 0, length = str.length; i < length; i++) {`